pkg/labels/labels_test.go

```diff
@@ -179,7 +207,7 @@ func (s *LabelsSuite) TestLabelSliceSHA256Sum(c *C) {
 	}
 	sha256sum, err := LabelSliceSHA256Sum(labels)
 	c.Assert(err, IsNil)
-	c.Assert(sha256sum, DeepEquals, "180d22655dec0e78f62c3e4ac17f5994baedbdfc5c882cbbd668e2628c44f320")
+	c.Assert(sha256sum, DeepEquals, "330d57790ccf8cd0b10000d6297baa5fa27b9d6e1cb228055b45dc6e440ea4da")
```

Comment:
It will assign a new security identity to the pod and will evaluate a new policy which will have the same effect. I don't think it breaks anything except that for a group of containers that previously used a single identity, they may use two identities for a while but with the same policy enforcement result.

Either way, let's merge it for 0.10 if you agree with the overall concept and we can think about a 0.9 backport separately.